src/list.html

```diff
@@ -100,6 +100,12 @@ <h2 class="subtitle">
               <li><a href="http://slack.memphistechnology.org/" target="_blank">memtech</a> - Memphis technical community chat.</li>
               <li><a href="https://nashdev.com" target="_blank">NashDev</a> - A network for Nashville developers.</li>
             </ul>
+	
+            <li><h3>Portland</h3></li>
```

Comment:
Let's go ahead and put Portland into the state of Oregon. Oregon now exists but you don't have that just yet. To get the new code do the following from the terminal inside your local directory:
```bash
git remote add upstream git@github.com:itzsaga/slack-list.git
git fetch upstream
git merge upstream/master master
```
Now you should have the most recent version of the project. Make your changes, add them, and commit them. They will automatically show up on the PR.

Just leave a comment when the changes are made and I'll review. 😃